apps/src/applab/dropletConfig.js

```diff
@@ -189,6 +189,7 @@ export var blocks = [
   {func: 'setStyle', parent: api, category: 'Advanced', params: ['"id"', '"color:red;"'] },
   {func: 'getAttribute', parent: api, category: 'Advanced', params: ['"id"', '"scrollHeight"'], type: 'value', noAutocomplete: true },
   {func: 'setAttribute', parent: api, category: 'Advanced', params: ['"id"', '"scrollHeight"', "200"], noAutocomplete: true},
+  {func: 'setSelectionRange', parent: api, category: 'Advanced', paletteParams: ['id', 'start', 'end'], params: ['"id"', '0', '0'], paramButtons: { minArgs: 3, maxArgs: 4 }},
```

Comment:
the getters don't need specific blocks because it's on the event, right?

Reply:
Correct - since they're just properties on the event object passed to event callbacks, adding blocks for them probably isn't very helpful.